package.json

```diff
@@ -43,6 +42,7 @@
     "core-js": "^2.5.3",
     "graphql": "^0.13.2",
     "graphql-tag": "^2.10.0",
+    "ionic": "^4.8.0",
```

Comment:
@wtrocki Is this needed in dependencies or could it be in dev dependencies?

Reply:
@darahayes Adopted version from #25